backend/packages/identity-scan/src/scan/events/identity/identityTimelineOperations.js

```diff
@@ -0,0 +1,91 @@
+const {
+  identity: { getIdentityTimelineCollection },
+} = require("@statescan/mongo");
+const {
+  getCurrentBlockTimestamp,
+  getSubIdentityDisplay,
+} = require("../../utils/unitConversion");
+const {
+  EVENT_METHOD: {
+    JUDGEMENT_GIVEN,
+    JUDGEMENT_REQUESTED,
+    JUDGEMENT_UNREQUESTED,
+    SUB_IDENTITY_ADDED,
+    SUB_IDENTITY_REMOVED,
+    SUB_IDENTITY_REVOKED,
+  },
+} = require("../../constants");
+const { getIdentityStorage } = require("../../utils/getIdentityStorage");
+
+async function setIdentityEventForTimeline(method, event, indexer) {
+  let identityEvent = {};
+  const eventData = event.data;
+  const accountId = eventData[0].toString();
+
+  identityEvent = await getIdentityStorage(accountId);
```

Comment:
We don't have to store the identity info to a identity timeline record. What we should save is:
1. The account
2. The event name
3. The event args